api/src/main/java/org/pmiops/workbench/api/FeaturedWorkspaceController.java

```diff
@@ -37,4 +42,18 @@ public ResponseEntity<WorkspaceResponseListResponse> getFeaturedWorkspaces() {
           "Not implemented in this environment: combine the results of getFeaturedWorkspacesConfig() and getPublishedWorkspaces() to generate the list of featured workspaces.");
     }
   }
+
+  /**
+   * Temporary method to backfill the Featured Workspace DB table with information from the Featured
+   * Workspaces Config JSON. This method will be deleted after the backfill process has been
+   * completed in all environments using curl.
+   *
+   * @return List of all Featured workspaces saved in database table featured_workspace
+   */
+  @Override
```

Comment:
Is it ok to keep it in this controller and add an admin authority tag

@AuthorityRequired({Authority.FEATURED_WORKSPACE_ADMIN})

Its easier to clean this up rather than workspace admin

Reply:
OK